src/open_inwoner/templates/pages/plans/list.html

```diff
@@ -51,7 +51,7 @@ <h1 class="h1">
                                 <div class="table__item--notification-danger">{% trans "Actie vereist" %}</div>
                             {% endif %}
                         </td>
-                        <td class="table__item">{% button text="" href=plan.get_absolute_url icon="arrow_forward" icon_outlined=True transparent=True %}</td>
+                        <td class="table__item">{% button text="plan_detail" hide_text=True href=plan.get_absolute_url icon="arrow_forward" icon_outlined=True transparent=True %}</td>
```

Comment:
I think this should be `text=plan_detail` (without quotes)

Reply:
For me it doesn't work without quotes, because it then means it's a variable but I need a string here.
When I have the quotes I see the following:

![image](https://user-images.githubusercontent.com/63058071/220622781-78b25a69-c39a-43d9-802e-d6d297e7e075.png)

Reply:
My example was bad, I meant `text=plan.title`, or something else useful that makes sense to an accessibility user.

Having literally aria-label="plan_detail" doesn't seem helpful.